### PR TITLE
Modal 

### PR DESCRIPTION
**Pull request by Gautam2305**
- made modal for component library

Screenshot (93)

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Good to merge

File: modal.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/0a707a45d9.js" crossorigin="anonymous"></script>
+    <title>Modal</title>
+</head>
+<body>
+    <div class="modal-container">
+        <section class="modal-text-container">
```
Comment (kjais1720) [inline modal.html:13]: the parent container should be given the `section` tag and `div` should be given to its children

File: stylesheets/Components/modal.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    max-width: 50%;
+    margin: auto;
+    background-color: var(--st-light-pink);
+    padding: 15px;
+}
+.modal-info, .modal-btn{
+    padding: 20px;
+}
+.btn-modal-solid{
+    padding: 15px;
```
Comment (kjais1720) [inline stylesheets/Components/modal.css:11]: try to use rems

**Review by kjais1720 — APPROVED** [review]
All good, just a couple of suggestions

File: stylesheets/Components/modal.css (hunk 2 above)
Comment (Gautam2305) [reply stylesheets/Components/modal.css:11]: root-font size is 20px..... and i also don't want the padding to be changed as it was looking odd when i used rems.

File: modal.html (hunk 1 above)
Comment (Gautam2305) [reply modal.html:13]: okay...will look into that....thank you for feedback

**Pull request merged**